index.js

```diff
@@ -102,7 +102,7 @@ function getSetSchemesData({
 
     return [
         avatar,
-        [ethers.utils.toUtf8Bytes('JoinAndQuit'), ethers.utils.toUtf8Bytes('FundingRequest'), ethers.utils.toUtf8Bytes('SchemeFactory')],
+        [ethers.utils.formatBytes32String('JoinAndQuit'), ethers.utils.formatBytes32String('FundingRequest'), ethers.utils.formatBytes32String('SchemeFactory')],
```

Comment:
looks like appropriate handling as the params should be bytes32.
Though wondering how the repo tests passed or what was the issue which require this change ? @markspereira @ben-kaufman

Reply:
The issue that required the change was that the utf-8 bytes values were causing an error when trying to write to the smart contract and upon changing to bytes32 it resolved the error

Reply:
might be .  Though how the repo test works without that fix ? see https://github.com/daostack/CommonFactory/blob/master/test/commonfactory.test.js#L76